hca/util/__init__.py

```diff
@@ -332,11 +336,37 @@ def _get_oauth_token_from_service_account_credentials(self):
         r.session.close()
         return credentials.token, credentials.expiry
 
+    def _get_jwt_from_service_account_credentials(self):
+        assert 'GOOGLE_APPLICATION_CREDENTIALS' in os.environ
+        service_account_credentials_filename = os.environ['GOOGLE_APPLICATION_CREDENTIALS']
+        if not os.path.isfile(service_account_credentials_filename):
+            msg = 'File "{}" referenced by the GOOGLE_APPLICATION_CREDENTIALS environment variable does not exist'
+            raise Exception(msg.format(service_account_credentials_filename))
+        with open(service_account_credentials_filename) as fh:
+            service_credentials = json.load(fh)
+
+        audience = "https://dev.data.humancellatlas.org/"
+        iat = time.time()
+        exp = iat + 3600
```

Comment:
Probably good to parameterize this through a kwarg to this method.

Reply:
will fix

Reply:
https://github.com/HumanCellAtlas/dcp-cli/pull/182